test/LibraryManager.Test/Providers/JsDelivr/JsDelivrCatalogTest.cs

```diff
@@ -17,10 +20,15 @@ public class JsDelivrCatalogTest
     {
         private static JsDelivrCatalog SetupCatalog(IWebRequestHandler webRequestHandler = null)
         {
+            var packageSearch = new NpmPackageSearch();
```

Comment:
These should probably follow the same pattern as the `webRequestHander` - optionally pass in a test fake, otherwise create a default.  That way they're easily used with an isolation framework (to generate a fake) and still able to reuse this method so that individual tests don't need to know/fake all the rest of the dependencies as well.

Reply:
Fixed with commit 9